src/server.c

```diff
@@ -5253,6 +5275,8 @@ const char *replstateToString(int replstate) {
     case SLAVE_STATE_WAIT_BGSAVE_START:
     case SLAVE_STATE_WAIT_BGSAVE_END:
         return "wait_bgsave";
+    case SLAVE_STATE_BACKGROUND_RDB_LOAD:
+        return "psync";
```

Comment:
how is that "partial sync" state?
```suggestion
        return "bg_transfer";
```
maybe `SLAVE_STATE_BACKGROUND_RDB_LOAD` should be `SLAVE_STATE_BG_TRANSFER`?
or maybe i'm misreading something...